File: myFlink/main/scala/lesson01/WordCount.scala
package lesson01

import org.apache.flink.streaming.api.scala._

object WordCount3 {
  def main(args: Array[String]): Unit = {
    val env = StreamExecutionEnvironment.getExecutionEnvironment
    val data = env.socketTextStream("192.168.123.153", 9988)
    data.flatMap(line => line.split(","))
      .map((_,1))
      .keyBy(0)
      .sum(1)
      .print()
    env.execute("wordcount")

  }

}

File: myFlink/main/java/flink/dataset/WordCount.java
package flink.dataset;

import org.apache.flink.api.java.ExecutionEnvironment;
import org.apache.flink.api.java.operators.DataSource;

public class WordCount {
    public static void main(String[] args) throws Exception {
        //一、获取离线程序入口
        ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
        String inputPath = "";
        //二、获取数据源
        DataSource<String> dataSet = env.readTextFile(inputPath);
        //三、数据处理
        //dataSet.map()

    }
}
